specification/network/resource-manager/Microsoft.Network/stable/2024-05-01/loadBalancer.json

```diff
@@ -1461,6 +1461,79 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{groupName}/providers/Microsoft.Network/loadBalancers/{loadBalancerName}/loadBalancingRules/{loadBalancingRuleName}/health": {
```

Comment:
The guidelines of using a POST api is that the action name should be a verb. 
How about "generateHealth" ? or something in that lines.